modules/admin-ui-frontend/app/scripts/shared/services/restServiceMonitor.js

```diff
@@ -58,11 +98,7 @@ angular.module('adminNg.services')
         services.service[AMQ_NAME].error = true;
       }
     }).catch(function(err) {
-      Monitoring.populateService(AMQ_NAME);
```

Comment:
What was the reasoning behind removing this?

Reply:
Monitoring.setError() already calls the Monitoring.populateService() function. 
Therefore I thought it would not be necessary to call it twice. 
Basically it was just to remove redundant code.